src/map/latent_effect.h

```diff
@@ -91,6 +91,7 @@ enum class LATENT : uint16
     VS_ECOSYSTEM          = 59, // Vs. Specific Ecosystem ID (e.g. Vs. Plantoid: Accuracy+3)
     VS_FAMILY             = 60, // Vs. Specific Family ID (e.g. Vs. Korrigan: Accuracy+3)
     VS_SUPERFAMILY        = 61, // Vs. Specific SuperFamily ID (e.g. Vs. Mandragora: Accuracy+3)
+    MAINJOB               = 62, // mainjob - PARAM: JOBTYPE
 };
 
 #define MAX_LATENTEFFECTID 61
```

Comment:
Doesn't this value also need to be updated?

Reply:
good catch, it does. And looks like it was missed previously or overwritten in prior pr also\*, it should be 63 as of this pr.

\* probably 2 pr's came in at same time and conflicted on this spot

Reply:
yes and no, I actually started it before I read this comment!
the value doesn't seem to be used anything but should be updated nonetheless

Reply:
For clarity's sake, I'm referring to Line 97, doesn't the MAX_LATENTEFFECTID value need updated?

Reply:
ah you are correct, it starts at 0 and was missed... I'll fix it to 63 then

Reply:
orrrrrr it technically says "max latent IDs" not "count" maybe it is supposed to be 62? maybe I should just delete it

Reply:
the way we use max IDs requires them to be 1+highest used ID.

Reply:
If you removed that max id line, the server would implode in odd places. its necessary.

Reply:
I guess we'll leave it at 63 and hope for the best

Reply:
63 is correct, so that is the best outcome.